java/ql/src/semmle/code/java/frameworks/FastJson.qll

```diff
@@ -0,0 +1,50 @@
+/**
+ * Provides classes and predicates for working with the FastJson framework.
+ */
+
+import java
+
+/**
+ * The class `com.alibaba.fastjson.JSON` or `com.alibaba.fastjson.JSONObject`.
+ */
+class FastJson extends RefType {
+  FastJson() {
+    this.hasQualifiedName("com.alibaba.fastjson", "JSON") or
```

Comment:
I believe `JSONObject` is a subclass of `JSON` so there should be no need for the `or`. There are also several other subclasses of `JSON` . See: https://github.com/alibaba/fastjson/tree/master/src/main/java/com/alibaba/fastjson

Reply:
I've removed `JSONObject`. It never really made sense to include, since the relevant methods are static.